newEraCaps/flows/fulfillment/README.MD

```diff
@@ -22,6 +22,10 @@ Once an order has been packed and is ready for pickup, an email to the customer
 
 New Era Caps is also rolling out a new locker based pickup solution where store staff place the packed order items into a locker where customers can pick them up at their own convenience. To support this workflow, staff will mark orders as packed in the BOPIS fulfillment app after placing the order items in the locker. The email sent to the customer will include a unique password and locker number which the customer can use to unlock the locker and collect their order items.
 
+### Not picked orders
```

Comment:
This is not very descriptive in helping the reader understand why this process exists. What kind of decision making was done and if re-stocking was even an option. These details should be considered when writing this section.